### DISCUSSION TITLE
Feature Request: Option to enable unlisted 7TV emotes

### DISCUSSION THREAD

**Discussion by ScrubN** [night/betterttv#5926] [Ideas]
An option to enable unlisted 7TV emotes would be nice. I'm not talking about marked as explicit, but rather emotes that have the [`listed` emote state boolean](https://github.com/SevenTV/Common/blob/4139fcc3eb8d79003573b26b552ef112ec85b8df/structures/v3/type.emote.go#L133-L134) set to false. This means either the author of the emote has not attempted to list it or the 7TV mods have not gotten around to verifying it yet.

Doing this seems to be as easy as adding an option bypass to the following lines:
Channel: https://github.com/EdVraz/betterttv/blob/136f34b62970005cd8ce2008854231147097e20b/src/modules/seventv/channel-emotes.js#L59-L62
Global: https://github.com/EdVraz/betterttv/blob/136f34b62970005cd8ce2008854231147097e20b/src/modules/seventv/global-emotes.js#L44-L47

Example of a SFW unlisted 7TV emote: https://7tv.app/emotes/6287c2ca6d9cd2d1f31b5e7d

**Comment by night** [comment]
We have no current plans to allow unlisted/removed emotes on our extension.
